env/WCOSS2.env

```
@@ -67,21 +67,13 @@ elif [[ "${step}" = "aeroanlvar" ]]; then
 
     export APRUNCFP="${launcher} -np \$ncmd ${mpmd_opt}"
 
-    nth_max=$((npe_node_max / npe_node_aeroanlvar))
-
-    export NTHREADS_AEROANL=${nth_aeroanlvar:-${nth_max}}
-    [[ ${NTHREADS_AEROANL} -gt ${nth_max} ]] && export NTHREADS_AEROANL=${nth_max}
-    export APRUN_AEROANL="${launcher} -n ${npe_aeroanlvar}"
+    export NTHREADS_AEROANL=${NTHREADSmax}
+    export APRUN_AEROANL="${APRUN}"
 
 elif [[ "${step}" = "aeroanlgenb" ]]; then
 
-    export APRUNCFP="${launcher} -np \$ncmd ${mpmd_opt}"
-
-    nth_max=$((npe_node_max / npe_node_aeroanlgenb))
-
-    export NTHREADS_AEROGENB=${nth_aeroanlgenb:-${nth_max}}
-    [[ ${NTHREADS_AEROGENB} -gt ${nth_max} ]] && export NTHREADS_AEROGENB=${nth_max}
-    export APRUN_AEROGENB="${launcher} -n ${npe_aeroanlgenb}"
+    export NTHREADS_AEROANLGENB=${NTHREADSmax}
+    export APRUN_AEROANLGENB="${APRUN} --cpus-per-task=${NTHREADS_AEROANLGENB}"
```

This job did not include threading flags for WCOSS2 before.  Are you sure it is needed?

If so, then this should probably be
```suggestion
    export APRUN_AEROANLGENB="${APRUN} -ppn ${tasks_per_node} --cpu-bind depth --depth ${NTHREADS_GSI}"
```

---

gotcha, no I don't think I need threading, so `${APRUN}` is sufficient, yes?

---

Yep, APRUN alone is sufficient in that case.
```suggestion
    export APRUN_AEROANLGENB="${APRUN}"
```